index.js

```diff
@@ -224,6 +230,27 @@ function init (ssb, config) {
       state.newAuthorListeners.push(fn)
     },
 
-    application: scuttle.application
+    application: scuttle.application,
+
+    subtribes: {
+      create (groupId, opts, cb) {
+        // create a new group
+        createGroup({}, (data) => {
```

Comment:
Hoping the dmKey isnt needed in the group/init message?

Reply:
and can just be generate here and returned eeek

Reply:
yeah the dmKey is just a random secret string!
just fake it with "secret string WOOO" for the moment

I think we're actually going to want a new message type where we publish these dmKeys.... hmmm but maybe that was on group profiles. It would be nice if it was independant of profiles so that tribes can just also watch for those messages